### ISSUE TITLE
debug: cannot stop running process (run without debugging)

### ISSUE THREAD

**Issue by xfan1024** [golang/vscode-go#165]
### What version of Go, VS Code & VS Code Go extension are you using?
- Run `go version` to get version of Go
	- go version go1.13.8 linux/amd64
- Run `code -v` or `code-insiders -v` to get version of VS Code or VS Code Insiders
	- 1.45.1/5763d909d5f12fe19f215cbfdd29a91c0fa9208a/x64
- Check your installed extensions to get the version of the VS Code Go extension 
	- 0.14.3
- Run `go env` to get the go development environment details
```plain
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/xiaofan/.cache/go-build"
GOENV="/home/xiaofan/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/xiaofan/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/lib/go-1.13"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/lib/go-1.13/pkg/tool/linux_amd64"
GCCGO="gccgo"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build345269875=/tmp/go-build -gno-record-gcc-switches"
```

### Share the Go related settings you have added/edited

Run `Preferences: Open Settings (JSON)` command to open your settings.json file.
Share all the settings with the `go.` or `["go"]` or `gopls` prefixes.

```json
{
    "go.formatTool": "goimports"
}
```

### Describe the bug
Running program using Ctrl+F5(Run Without Debugging), then I stop running process by hit stop button, but the process not be killed

My vscode running in Windows 10 version 2004, and use Remote-WSL pluggin connect to Ubuntu-20.04(WSL2)

**Comment by hyangah** [comment]
@xfan1024 Thanks for the report. Can you share your `launch.json` if exists? 

@quoctruong Can you help reproduce the problem with Remote-WSL? 

I tried the following program with no `launch.json` but failed to reproduce the problem in local machine. So, I am guessing this is related to the remote setup.

```
package main

import (
	"fmt"
	"time"
)

func main() {
	start := time.Now()
	for time.Since(start) < 1*time.Hour {
		fmt.Println(time.Now())
		time.Sleep(1 * time.Second)
	}
}
```

**Comment by xfan1024** [comment]
@hyangah 
this is my launch.json
``` js
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch GO",
            "type": "go",
            "request": "launch",
            "mode": "auto",
            "program": "${fileDirname}",
            "env": {},
            "args": []
        }
    ]
}

```

**Comment by hyangah** [comment]
@xfan1024 Thanks for sharing the `launch.json` configuration - looking at it now, this is not the `remote` mode debugging discussed [here](https://github.com/golang/vscode-go/blob/master/docs/debugging.md#remote-debugging), but a local debug mode running on the remote host. I am guessing it's a duplicate of #185 and should be fixed in the next release.

/cc @polinasok

**Comment by hyangah** [comment]
Fix is available since 0.15.0
